src/PlayState.cpp

```diff
@@ -1,12 +1,17 @@
 #include <memory>
 #include <chrono>
 #include <cmath>
+#include <algorithm>
 #include <PlayState.hpp>
 #include <GfxStream.hpp>
 #include <MenuState.hpp>
 #include <View/GameView.hpp>
 
-PlayState::PlayState(int boardSize) : board(Board(boardSize)), x(0), y(0) {}
+PlayState::PlayState(int boardSize) : board(Board(boardSize)), x(0), y(0) {
```

Comment:
Po co w modelu trzymać coś co wpływa tylko na to jak to jest wyświetlane?

Reply:
to nie jest od wyświetlania? zoabcz na handleinput